TeePod/modules/Result/ResultPresenter.swift

```diff
@@ -33,7 +33,28 @@ final class ResultPresenter: ObservableObject {
     }
 }
 
-extension ResultPresenter {}
+extension ResultPresenter {
+    func getComment() -> String {
+        var comment: String
+        if tiredness! <= 60 {
+            comment = "時間が延長されました"
+        } else {
+            comment = "適度に休憩を取りましょう"
+        }
+        return comment
+    }
```

Comment:
こんな感じに書くと短くなってもっと読みやすくなるよ！🐹
```suggestion
    func getComment() -> String {
        if tiredness! <= 60 {
            return "時間が延長されました"
        }
        return "適度に休憩を取りましょう"
    }
```

Reply:
あい！！！！！！！！！！！！！！